lms/djangoapps/grades/tests/test_models.py

```diff
@@ -204,13 +204,9 @@ def test_update(self):
     def test_save(self, already_created):
         if already_created:
             _ = PersistentSubsectionGrade.create(**self.params)
-            mock_fn_called = "update"
-            mock_fn_uncalled = "create"
-        else:
-            mock_fn_called = "update"
-            mock_fn_uncalled = "create"
-        with patch("lms.djangoapps.grades.models.PersistentSubsectionGrade." + mock_fn_called) as mock_called:
-            with patch("lms.djangoapps.grades.models.PersistentSubsectionGrade." + mock_fn_uncalled) as mock_uncalled:
-                PersistentSubsectionGrade.save_grade()
-                self.assertTrue(mock_called.called)
-                self.assertFalse(mock_uncalled.called)
+        module_prefix = "lms.djangoapps.grades.models.PersistentSubsectionGrade."
+        with patch(module_prefix + "create") as mock_create:
+            with patch(module_prefix + "update", wraps=PersistentSubsectionGrade.update) as mock_update:
+                PersistentSubsectionGrade.save_grade(**self.params)
+                self.assertTrue(mock_update.called)
+                self.assertNotEqual(mock_create.called, already_created)
```

Comment:
Ah.  This makes sense.  It calls update either way, and create only if it doesn't exist yet.